lms/djangoapps/notes/tests.py

```diff
@@ -39,11 +42,53 @@ def test_notes_enabled(self):
         Tests that notes are enabled when the course tab configuration contains
         a tab with type "notes."
         '''
-        self.course.tabs = [{'type': 'foo'},
-                            {'name': 'My Notes', 'type': 'notes'},
-                            {'type': 'bar'}]
+        with self.settings(FEATURES={'ENABLE_STUDENT_NOTES': True}):
```

Comment:
It looks like I missed this when I switched the ``is_enabled`` code over to using Django settings directly. I wonder how this still works...